dapps/marketplace/src/components/AttestationBadges.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+
+const AttestationBadges = ({
+  className,
+  providers,
+  minCount = 10,
+  fillToNearest = 5,
+  onClick
+}) => {
+  let badges = providers || []
+
+  if (badges.length < minCount) {
+    badges = badges.concat(new Array(minCount - badges.length).fill(null))
+    // Show a minimum of `minCount` icons
+  } else if (badges.length > minCount && badges % fillToNearest !== 0) {
```

Comment:
This should probably be: `else if (badges.length > minCount && badges.length % fillToNearest !== 0)` see the extra `length`

Reply:
Nice catch. Will fix it.